tests/php/src/Validation/URLValidationProviderTest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace AmpProject\AmpWP\Tests\Validation;
+
+use AmpProject\AmpWP\Tests\Helpers\AssertContainsCompatibility;
+use AmpProject\AmpWP\Tests\Helpers\PrivateAccess;
+use AmpProject\AmpWP\Tests\Helpers\ValidationRequestMocking;
+use AmpProject\AmpWP\Validation\URLValidationProvider;
+use WP_UnitTestCase;
+
+/** @coversDefaultClass URLValidationProvider */
+final class URLValidationProviderTest extends WP_UnitTestCase {
+	use PrivateAccess, AssertContainsCompatibility;
+
+	/**
+	 * Setup.
+	 *
+	 * @inheritdoc
+	 */
+	public function setUp() {
+		parent::setUp();
+		$this->validation_provider = new URLValidationProvider( 100 );
```

Comment:
`URLValidationProvider` does not have a constructor, so the `100` does not do anything here.
```suggestion
		$this->validation_provider = new URLValidationProvider();
```